src/sentry/apidocs/hooks.py

```diff
@@ -111,21 +111,28 @@ def __write_ownership_data(ownership_data: Dict[ApiOwner, Dict]):
     processed_data = {}
     index = 2
     for team in ownership_data:
+        # sorting APIs list so it doesn't trigger file change on every commit
+        public_apis = list(ownership_data[team][ApiPublishStatus.PUBLIC])
+        public_apis.sort()
+        private_apis = list(ownership_data[team][ApiPublishStatus.PRIVATE])
+        private_apis.sort()
+        experimental_apis = list(ownership_data[team][ApiPublishStatus.EXPERIMENTAL])
+        experimental_apis.sort()
+        unknown_apis = list(ownership_data[team][ApiPublishStatus.EXPERIMENTAL.UNKNOWN])
+        unknown_apis.sort()
```

Comment:
shorthand sorting + casting to list
```suggestion
        public_apis = sorted(ownership_data[team][ApiPublishStatus.PUBLIC])
        private_apis = sorted(ownership_data[team][ApiPublishStatus.PRIVATE])
        experimental_apis = sorted(ownership_data[team][ApiPublishStatus.EXPERIMENTAL])
        unknown_apis = sorted(ownership_data[team][ApiPublishStatus.EXPERIMENTAL.UNKNOWN])
```